controller/userController.js

```diff
@@ -0,0 +1,23 @@
+const { validationResult } = require("express-validator");
+const { registerHelper, loginHelper } = require("../helper/userHelper");
+
+const userRegister = async (req, res) => {
+  try {
+    const errors = validationResult(req);
+    if (!errors.isEmpty()) {
+      return res.status(400).json({ errors: errors.array() });
+    }
```

Comment:
Handle this in validationMiddleware

Reply:
No need of validationresult as im using regex